src/python/gudhi/persistence_graphical_tools.py

```diff
@@ -20,6 +20,20 @@
 __copyright__ = "Copyright (C) 2016 Inria"
 __license__ = "MIT"
 
+@lru_cache(maxsize=1)
+def _matplotlib_can_use_tex():
+    """This function returns True if matplotlib can deal with LaTeX, False otherwise.
+    The returned value is cached.
+    """
+    try:
+        from matplotlib import checkdep_usetex
+        return checkdep_usetex(True)
+    except ImportError:
+        print("This function is not available, you may be missing matplotlib.")
+
+
+_gudhi_matplotlib_use_tex = _matplotlib_can_use_tex()
```

Comment:
The message "This function is not available, you may be missing matplotlib." will appear as soon as someone does `import gudhi`, since the module is imported by default.

Reply:
Ok, so let's do it differently on 37f80627